README.md

```diff
@@ -54,14 +54,14 @@ $ mkdir tutorial
 $ cd tutorial
 $ mkdir rootfs
 $ docker export $(docker create busybox) | tar -C rootfs -xvf -
-// Prepare a configuration file for the container that will run `sleep 5`.
+$ # Prepare a configuration file for the container that will run `sleep 5`.
```

Comment:
There are only comments in this line, have you any reason to put `$` at the beginning?

Reply:
I think sometimes confuse `stdout` when not add `$` mark ahead (or execute on root `#`).

Another approach is to remove new line and join before command.
Like this, 
```sh
$ docker export $(docker create busybox) | tar -C rootfs -xvf -  # Prepare a configuration file for the container that will run `sleep 5`.
```
Which do you like? I think both okay.

Reply:
That's certainly a good perspective.
> I think sometimes confuse stdout when not add $ mark ahead (or execute on root #).

I think it's fine as it is now.  If you don't have time, this PR will be very good to merge as it is.
If you like, how about dividing some commands into meaningful units (e.g. build/prepare for container execution / execute container)? This way you could add a light description to each separated section to erase this commented-out description.

Reply:
Thank you for your suggestion. Okay, I change to separate some section. Please wait a moment.